### ISSUE TITLE
Where are advanced scripting code and BPA rules stored

### ISSUE THREAD

**Issue by DKfromtheBay** [otykier/TabularEditor#868]
I will be getting a new PC soon (switching companies) and have lots of custom code snippets and BPA rules set up.  Is it correct that the JSON files in the folder: Local Disk --> Users --> me --> AppData --> Local --> TabularEditor are the ones I should backup so I can quickly get set up on my new machine?

![image](https://user-images.githubusercontent.com/78566309/121791735-f5444f80-cba1-11eb-879c-ac60920c443c.png)

**Comment by m-kovalsky** [comment]
Correct, this is documented here: https://docs.tabulareditor.com/Best-Practice-Analyzer.html#best-practice-analyzer
